sql/src/main/java/io/crate/metadata/blob/BlobTableInfo.java

```diff
@@ -155,10 +155,13 @@ public ColumnIdent clusteredBy() {
     }
 
     private void registerStaticColumns() {
+        int pos = 0;
```

Comment:
could be Integer to avoid boxing

Reply:
I think due to the `pos++` there would be boxing with `Integer pos` as well - looking at the bytecode of both variant it seems even worse.

So I'd rather stick with `int pos` to make it clear that it can't be null

Reply:
ok didn't know that, thx.